pkg/datasources/security_integrations_acceptance_test.go

```diff
@@ -8,24 +8,521 @@ import (
 	acc "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance"
 
 	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/helpers/random"
-	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/testenvs"
 	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/provider/resources"
 	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
 	"github.com/hashicorp/terraform-plugin-testing/config"
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
 	"github.com/hashicorp/terraform-plugin-testing/tfversion"
 )
 
```

Comment:
can we add one more test (with not that many assertions) that shows that it works when multiple different type integrations are created, pls?

Reply:
Done. I couldn't find a way to assert that output is sorted, but it seems like snowflake show output is always sorted by integration type. I tried this a few times locally and it looks like this order should be consistent.

Reply:
We will observe and adjust if needed.